pages/css/caseHeaders.css

```diff
@@ -0,0 +1,43 @@
+.navigation-tabs {
+    border-bottom: 1px solid #ddd;
+    padding-left: 0;
+    margin-bottom: 0;
+    list-style: none;
+    display: table;
+}
+
+.navigation-tabs--item {
+    margin-bottom: -3px;
+    cursor: pointer;
+    float: left;
+    position: relative;
+    display: block;
+}
+
+.navigation-tabs--item > a:hover,
```

Comment:
I think it's probably worth introducing a class for the link and then not requiring a strict child relationship.

I've been trying to unpick the direct child link on the tabulation actions recently and its a pain

Reply:
Done